WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -145,16 +157,42 @@ final class DashboardViewController: UIViewController {
         return true
     }
 
-    /// Hide the announcement card when the navigation bar is compact
-    ///
-    func updateAnnouncementCardVisibility() {
-        announcementView?.isHidden = navigationBarIsShort
+    func showHeader() {
```

Comment:
On one hand, I like the idea of reducing duplication, but I think I lean towards the current implementation since it seems clearer what showing or hiding would do, without having to parse inline conditionals.